cargo-shuttle/src/lib.rs

```diff
@@ -2057,20 +2060,27 @@ fn check_version(runtime_path: &Path) -> Result<()> {
     let my_version = semver::Version::from_str(VERSION).unwrap();
 
     if !runtime_path.try_exists()? {
-        bail!("shuttle-runtime is not installed");
+        bail!("shuttle-runtime binary not found");
     }
 
     // Get runtime version from shuttle-runtime cli
-    let runtime_version = std::process::Command::new(runtime_path)
-        .arg("--version")
-        .output()
-        .context("failed to run the shuttle-runtime binary to check its version")?
-        .stdout;
+    // It should print the version and exit immediately, so a timeout is used to not launch servers with non-Shuttle setups
```

Comment:
Isn't the process exiting immediately already when running the runtime with `--version`?